### ISSUE TITLE
Allow users to set the account for the derivation path in the installer

### ISSUE THREAD

**Issue by darosior** [wizardsardine/liana#291]

**Comment by edouardparis** [comment]
#289 directly select accounts for users in the create wallet panel and let user generate multiple key with incremented indexes in the participate panel.

**Comment by darosior** [comment]
Yeah, let's keep tracking this for power users that want to set the account themselves because, say, they are using this signing device at this derivation path elsewhere already.

**Comment by kloaec** [comment]
NACK for the general case, ACK for an advanced option/display.
I would use this feature personally.
